src/test/ui/associated-types/issue-65774-2.stderr

```diff
@@ -9,6 +9,11 @@ LL |     type MpuConfig: MyDisplay = T;
 error[E0277]: the trait bound `T: MyDisplay` is not satisfied
   --> $DIR/issue-65774-2.rs:16:6
    |
+LL | trait MPU {
+   |       ---
```

Comment:
in any case I feel like I would like a label here that says "bound declared in this trait" or something

Reply:
Would "required by a bound in this" be good?